src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -434,6 +438,44 @@ private IOperation CreateBoundCallOperation(BoundCall boundCall)
             return new CSharpLazyInvocationOperation(this, boundCall, targetMethod.GetPublicSymbol(), isVirtual, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
+        private IOperation CreateBoundFunctionPointerInvocationOperation(BoundFunctionPointerInvocation boundFunctionPointerInvocation)
+        {
+            if (boundFunctionPointerInvocation.HasErrors)
+            {
+                return new CSharpLazyInvalidOperation(
+                    this,
+                    boundFunctionPointerInvocation,
+                    _semanticModel,
+                    boundFunctionPointerInvocation.Syntax,
+                    boundFunctionPointerInvocation.Type.GetPublicSymbol(),
+                    ConvertToOptional(boundFunctionPointerInvocation.ConstantValue),
+                    isImplicit: boundFunctionPointerInvocation.WasCompilerGenerated);
+            }
+
+            return new CSharpLazyInvocationOperation(
+                this,
+                boundFunctionPointerInvocation,
+                boundFunctionPointerInvocation.FunctionPointer.Signature.GetPublicSymbol(),
+                isVirtual: false,
```

Comment:
>isVirtual: false, [](start = 16, length = 17)

We don't know that, right?

Reply:
We do. Function pointers do not `callvirt`. The method is not dynamically resolved based on the vtable.

---
In reply to: [428790957](https://github.com/dotnet/roslyn/pull/44436#discussion_r428790957) [](ancestors = 428790957)

Reply:
> The method is not dynamically resolved based on the vtable.

We don't know that. This is part of the pointer. 

---
In reply to: [428829524](https://github.com/dotnet/roslyn/pull/44436#discussion_r428829524) [](ancestors = 428829524,428790957)